ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/components/ReportParametersDetails.tsx

```diff
@@ -123,22 +123,35 @@ function ReportParametersDetails({
                             {getCVEsDiscoveredSinceText(formValues.reportParameters)}
                         </DescriptionListDescription>
                     </DescriptionListGroup>
+                    <DescriptionListGroup>
+                        <DescriptionListTerm>Non-optional columns</DescriptionListTerm>
+                        <DescriptionListDescription>Cluster</DescriptionListDescription>
+                        <DescriptionListDescription>Namespace</DescriptionListDescription>
+                        <DescriptionListDescription>Deployment</DescriptionListDescription>
+                        <DescriptionListDescription>Image</DescriptionListDescription>
+                        <DescriptionListDescription>Component</DescriptionListDescription>
+                        <DescriptionListDescription>CVE</DescriptionListDescription>
+                        <DescriptionListDescription>Fixable</DescriptionListDescription>
+                        <DescriptionListDescription>CVE Fixed In</DescriptionListDescription>
+                        <DescriptionListDescription>Severity</DescriptionListDescription>
+                        <DescriptionListDescription>CVSS</DescriptionListDescription>
+                        <DescriptionListDescription>Discovered At</DescriptionListDescription>
+                        <DescriptionListDescription>Reference</DescriptionListDescription>
+                    </DescriptionListGroup>
                     {(hasIncludeNvdCvss || hasIncludeEpssProbability || hasIncludeAdvisory) && (
                         <DescriptionListGroup>
                             <DescriptionListTerm>Optional columns</DescriptionListTerm>
                             {hasIncludeNvdCvss && (
-                                <DescriptionListDescription>
-                                    Include NVD CVSS
-                                </DescriptionListDescription>
+                                <DescriptionListDescription>NVDCVSS</DescriptionListDescription>
```

Comment:
Ouch, I was too busy to notice at the time, and now a breaking change to fix in CSV heading row.